avocado/utils/crypto.py

```diff
@@ -57,9 +40,10 @@ def hash_file(filename, size=None, algorithm="md5"):
         size = fsize
 
     try:
-        hash_obj = hash_wrapper(algorithm=algorithm)
-    except ValueError:
-        logging.error("Unknown hash algorithm %s, returning None", algorithm)
+        hash_obj = hashlib.new(algorithm)
```

Comment:
with this you should also modify the `supported methods are md5 (default) or sha1` docstring. Perhaps just mention by default `md5`.